beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/filter/WhitelistedListenerEventFilter.java

```diff
@@ -31,10 +31,12 @@
   private static final String BEEKEEPER_HIVE_EVENT_WHITELIST = "beekeeper.hive.event.whitelist";
 
   @Override
-  public boolean filter(ListenerEvent listenerEvent, LifecycleEventType lifecycleEventType) {
+  public boolean isFilteredOut(ListenerEvent listenerEvent, LifecycleEventType lifecycleEventType) {
     Map<String, String> tableParameters = listenerEvent.getTableParameters();
-    if (tableParameters != null && tableParameters.get(BEEKEEPER_HIVE_EVENT_WHITELIST) != null) {
-      return !isWhitelisted(listenerEvent, tableParameters.get(BEEKEEPER_HIVE_EVENT_WHITELIST));
+    if (tableParameters != null
+        && tableParameters.get(WhitelistedListenerEventFilter.BEEKEEPER_HIVE_EVENT_WHITELIST) != null) {
```

Comment:
This can go: `WhitelistedListenerEventFilter.` and the same on line 39.

I think also we should rename this class and the table parameter. Perhaps we can do it as part of another ticket, but we should no longer make references to a "whitelist". Let's call this "allowlist" instead.

Reply:
+1 for renaming to AllowList

Reply:
Will raise a ticket